internal/errors/info.go

```diff
@@ -0,0 +1,47 @@
+package errors
+
+type Info struct {
+	Kind    Kind
+	Message string
+}
+
+// errorCodeInfo provides a map of unique Codes (IDs) to their
```

Comment:
I sort of didn't know the best place to put errCodeInfo since it's tightly coupled to Kind and Code... so I decide to just separate it from both of them.  We can always combine them if there seems to be a problem with drift.